UltiSnips/markdown.snippets

```diff
@@ -50,6 +50,23 @@ snippet spar "Paragraph" b
 $0
 endsnippet
 
+###################
+# Text formatting #
+###################
+
+snippet * "cursive"
```

Comment:
Would a letter based trigger be better? These are normally easier to type.

Would "italics" be a better description? It doesn't output cursive text.

Reply:
I like the asterisk(s) as a trigger. This way the user does not need to remember another trigger.

I agree. "italics" would be a better description. I corrected it in b8f54d395e2f0b20b0ae12e3e21dddad2a0b3fda.

It seems that the concrete formatting is indeed not specified in the commonmark specification. However the descriptions I used seem to describe well the concrete "effect" of the asterisks in common implementations (at least those I used).